### PR TITLE
Bolt4 copy edit

### PR DESCRIPTION
**Pull request by toadlyBroodle**
-Separate out requirements into their own sections.
-Add a couple [FIXME:] tags.

### CODE REVIEW

File: 04-onion-routing.md [hunk 1]
```diff
@@ -633,161 +633,182 @@ channel.
...
 1. type: 21 (`expiry_too_far`)
 
-If the channel is disabled, the current channel setting for the outgoing
-channel are reported:
 1. type: UPDATE|20 (`channel_disabled`)
 2. data:
    * [`2`: `flags`]
    * [`2`:`len`]
    * [`len`:`channel_update`]
+[FIXME: is it important that 20 and 21 remain in this order or can they be moved to end of list?]
```
Comment (rustyrussell) [inline 04-onion-routing.md]: Order can (and should!) be shuffled into logical ordering.  Numbers themselves are mainly relics of history, but hard to change.

File: 04-onion-routing.md [hunk 2]
```diff
@@ -633,161 +633,182 @@ channel.
...
+    - SHOULD fail the HTLC.
+    - SHOULD return an `incorrect_payment_amount` error.
+    - Note: this allows the origin node to reduce information leakage by
+    altering the amount while not allowing for accidental gross overpayment.
+  - if the `cltv_expiry` value is too near the present:
+    - MUST fail the HTLC.
+    - MUST return a `final_expiry_too_soon` error.
+  - if the `outgoing_cltv_value` does NOT correspond with the `cltv_expiry` from
+  the final node's HTLC:
+    - [FIXME: MAY|SHOULD|MUST?] return `final_incorrect_cltv_expiry` error.
```
Comment (rustyrussell) [inline 04-onion-routing.md]: MUST

File: 04-onion-routing.md [hunk 3]
```diff
@@ -633,161 +633,182 @@ channel.
...
+  - if the `amt_to_forward` is greater than the `incoming_htlc_amt` from the
+  final node's HTLC:
+    - [FIXME: MAY|SHOULD|MUST?] return a `final_incorrect_htlc_amount` error.
```
Comment (rustyrussell) [inline 04-onion-routing.md]: MUST

File: 04-onion-routing.md [hunk 4]
```diff
@@ -730,7 +733,7 @@ An _erring node_:
   - MUST select one of the above error codes when creating an error message.
   - MUST include the appropriate data for that particular error type.
   - if there is more than one error:
-    - SHOULD select the first error it encounters in list above.
+    - SHOULD select the first error it encounters from list above.
```
Comment (rustyrussell) [inline 04-onion-routing.md]: from the list above?

Comment (toadlyBroodle) [reply 04-onion-routing.md]: fixed

File: 04-onion-routing.md (hunk 1 above)
Comment (toadlyBroodle) [reply 04-onion-routing.md]: Ok, done.

File: 04-onion-routing.md (hunk 2 above)
Comment (toadlyBroodle) [reply 04-onion-routing.md]: done

File: 04-onion-routing.md (hunk 3 above)
Comment (toadlyBroodle) [reply 04-onion-routing.md]: ditto

**Comment by shannona** [comment]
On my first thing list for tomorrow.

**Pull request merged**